src/app/pages/dashboard/line-chart/line-chart.component.ts

```diff
@@ -1,267 +1,56 @@
 import * as d3 from 'd3';
 import * as nv from 'nvd3';
 
-import { Component, OnInit } from '@angular/core';
+import { Component, ElementRef, HostBinding, OnInit } from '@angular/core';
+
+import { LineChartComponent as BaseLineChartComponent } from 'theme/components/line-chart';
+
+import { LineChartService } from './line-chart.service';
 
 @Component({
   selector: 'app-line-chart',
-  styleUrls: ['./line-chart.component.scss'],
-  templateUrl: './line-chart.component.html',
+  styleUrls: ['../../../../theme/components/line-chart/line-chart.component.scss'],
```

Comment:
Is it possible to use more pretty path?

Reply:
No, https://github.com/angular/angular-cli/issues/5930#issuecomment-299428834

Only tricks with `require`/`import`